index.html

```diff
@@ -1 +1,44 @@
-Brendan O'Connor
+<h1>Brendan O'Connor</h1>
+
+<h2>About</h2>
+<!--paragraph about yourself-->
+<p>
+  I am from the blue mountains of the Shenandoah Valley and have a deep-running
+  love for American and Irish folk music, solidarity politics, breakfasts of all
+  kinds, and economix (i.e., economics done differently, like in
+  cooperatively-owned businesses). As I lived in Nashville, TN and built
+  relationships with other Southerners, I increasingly felt a sense of place and
+  purpose in the South. In 2016, I made a home in Selma, AL, where there are
+  good folks and there's good work to be done. My wife and I moved to Durham in
+  Nov. of 2021 with our new AL puppy, Inis, to start a next season closer to
+  some family and friends.
+</p>
+
+<h2>Experience</h2>
+<!--write a list of places you've worked or projects you've worked on -->
+<p>
+  I am new to coding in many ways, though I have always been intrigued by
+  technology. My Mom tells me that as a baby, she thought I might go into
+  computer-related work one day. For me, my first formal involvement in
+  tech-related work came through the cooperative sector--specifically through
+  something called "platform cooperatives," where workers and users own the tech
+  companies they make possible. I took a class through The New School and
+  Mondgragon University in 2020 called "Platform Cooperatives Now." This led me
+  into a volunteer project seeking to develop a co-op food delivery app. I was
+  more involved in coordinating the volunteer software developers among us, but
+  it gave me a taste for the grounded quality of building code and the role
+  coding can play in a modern economy. I since applied and was accepted into
+  Code the Dream in early 2022, where I am learning JavaScript, HTML, CSS, and
+  related introductory skills, which will be followed by a course in Fall of
+  2022 in either React.js or Node.js.
+</p>
+
+<h2>Connect</h2>
+<!--paste links to your email, GitHub, social media, etc. -->
+<p>Find me online on:</p>
+<ul>
```

Comment:
hi=) nice job, do you know what is the difference between <h2></h2> and <p></p> ? 
 a h2 is a block level element.

Reply:
Hey @Estheryu991 , sorry i think your 2nd Q here didn't come through clearly to me, would you mind restating it? Definitely hear you on the hierarchy of h1, h2, etc., thanks a lot for clarifying that and it's more logical purpose, opposed to formatting! Just didn't quite get your 2nd Q there...also apologies for the slower response!